packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -252,9 +252,18 @@ async function fetchLatestSnapshotCore(
                 });
 
                 let parsedSnapshotContents: IOdspResponse<ISnapshotContents> | undefined;
-
+                const splittedContentType = contentType?.split(";");
+                let contentTypeToRead: string | undefined;
+                if (splittedContentType !== undefined) {
+                    for (let type in splittedContentType) {
+                        if (type === "application/json" || type === "application/ms-fluid") {
+                            contentTypeToRead = type;
```

Comment:
Isn't it guaranteed to be a first item in the list?
Do we need to also break based on ","? I thought we can see "application/ms-fluid, application/ms-fluid" or something like that.
Another way to do it (seems simpler):
```ts
contentType?.indexOf("application/ms-fluid") !== -1
```
Avoids any issues with formatting,